src/pybind/mgr/prometheus/module.py

```diff
@@ -846,9 +849,9 @@ def config_notify(self) -> None:
         self.log.info('Restarting engine...')
         cherrypy.engine.stop()
         cherrypy.server.httpserver = None
+        server_addr = cast(str, self.get_localized_module_option('server_addr', get_default_addr()))
```

Comment:
In case if cephadm isn’t enabled previous implementation should still exist. Basically it’s all abt checking if orch.status() available or not and backend == cephadm

Reply:
Just for reference to get orch status https://github.com/ceph/ceph/blob/e580e3c681c594e9bf4238c9216288921eceb25b/src/pybind/mgr/dashboard/controllers/rbd_mirroring.py#L618

Reply:
> @rkachach ^^

Thanks @avanthakkar I'll add the check